prov/verbs/src/verbs_info.c

```diff
@@ -902,6 +960,8 @@ static int fi_ibv_get_srcaddr_devs(struct fi_info **info)
 	}
 
 	for (fi = *info; fi; fi = fi->next) {
+		if (fi->ep_attr->type == FI_EP_DGRAM)
+			continue;
```

Comment:
Is this not required for verbs/UD?

Reply:
Never mind. Just realized UD case is being handled in a separate function.

Reply:
Yes, it's correct